### ISSUE TITLE
Go modules: Broken go mod tidy

### ISSUE THREAD

**Issue by hasueki** [IBM-Cloud/terraform-provider-ibm#2872]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* N/A
### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please share a link to the ZIP file.
```

### Expected Behavior

Running `go mod tidy` should succeed without errors.

### Actual Behavior

```
$ go mod tidy
go: finding module for package github.com/IBM/apigateway-go-sdk/apigatewaycontrollerapiv1
go: found github.com/IBM/apigateway-go-sdk/apigatewaycontrollerapiv1 in github.com/IBM/apigateway-go-sdk v0.0.0-20210714141226-a5d5d49caaca
go: finding module for package github.com/IBM/apigateway-go-sdk
github.com/IBM-Cloud/terraform-provider-ibm/ibm imports
	github.com/IBM/apigateway-go-sdk: module github.com/IBM/apigateway-go-sdk@latest found (v0.0.0-20210714141226-a5d5d49caaca), but does not contain package github.com/IBM/apigateway-go-sdk
```

### Steps to Reproduce

1. `go mod tidy`

### References

* #0000

**Comment by yselkowitz** [comment]
I think this ended up being fixed by #2887.

**Comment by hkantare** [comment]
closing this issue
